spark/src/main/java/org/apache/zeppelin/spark/dep/SparkDependencyContext.java

```diff
@@ -64,6 +67,8 @@ public SparkDependencyContext(String localRepoPath, String additionalRemoteRepos
   }
 
   public Dependency load(String lib) {
+    Console.println("DepInterpreter(%dep) deprecated. "
```

Comment:
I used `Console` to print deprecated message in paragraph through scala compiler.
Can we keep it as it is?